edgekv/lib/edgekv.js

```diff
@@ -195,7 +203,8 @@ export class EdgeKV {
 			(response) => response.text(),
```

Comment:
Missing the retry option for this method.

Reply:
not needed for put or delete

Reply:
makes sense